### PR TITLE
[6]Develop a Products Page using HMML

### PR DESCRIPTION
**Pull request by ChabuChabu**
## Task: `Created a products page`
- `Created products page with two images and its description
product page

### CODE REVIEW

File: submissions/chabuChabu/html/product.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" >
+        <title>Products-page</title>
+    </head>
+    <body >
+
```
Comment (adaugochi) [inline submissions/chabuChabu/html/product.html:8]: Remove new line space

File: submissions/chabuChabu/html/product.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+        <h1>Kwathu</h1><hr>
+
+         <div >
+         	<h3>SAMSUNG  2020 73"Q85T QLED 4K HDR</h3>
+         </div>
+
+
```
Comment (adaugochi) [inline submissions/chabuChabu/html/product.html:16]: same here

File: submissions/chabuChabu/html/product.html [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+                
+         <table >
+            
+             <tr >
+
```
Comment (adaugochi) [inline submissions/chabuChabu/html/product.html:21]: same here

File: submissions/chabuChabu/html/product.html [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+         <table >
+            
+             <tr >
+                
+                
+                <td>
+                    <img src="product-01.jpg"><br>
+      
+                </td>
+
```
Comment (adaugochi) [inline submissions/chabuChabu/html/product.html:27]: same here

File: submissions/chabuChabu/html/product.html [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+      
+                </td>
+                
+                
+                 <td>
+                     <img src="product-02.jpg"><br>
+                  
+                </td>
+                 
+
```
Comment (adaugochi) [inline submissions/chabuChabu/html/product.html:34]: same here

File: submissions/chabuChabu/html/product.html [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+                     <li><a href="#">Samsung 47" SmartTV</a></li>
+                     <li><a href="#">Samsung 57" SmartTV</a></li>
+                     <li><a href="#">Samsung 60" SmartTV</a></li>
+                   </ol>
+             
+                 </td> 
+
+              </tr>
+         </table>
+        <!--<footer>
```
Comment (ToluAkin) [inline submissions/chabuChabu/html/product.html:89]: Remove this unused comments here @ChabuChabu

File: submissions/chabuChabu/html/product.html (hunk 1 above)
Comment (ChabuChabu) [reply submissions/chabuChabu/html/product.html]: Noted

File: submissions/chabuChabu/html/product.html (hunk 2 above)
Comment (ChabuChabu) [reply submissions/chabuChabu/html/product.html]: Noted

File: submissions/chabuChabu/html/product.html (hunk 3 above)
Comment (ChabuChabu) [reply submissions/chabuChabu/html/product.html]: Noted

File: submissions/chabuChabu/html/product.html (hunk 4 above)
Comment (ChabuChabu) [reply submissions/chabuChabu/html/product.html]: Noted

File: submissions/chabuChabu/html/product.html (hunk 5 above)
Comment (ChabuChabu) [reply submissions/chabuChabu/html/product.html]: Noted

File: submissions/chabuChabu/html/product.html (hunk 6 above)
Comment (ChabuChabu) [reply submissions/chabuChabu/html/product.html]: Noted

**Pull request merged**